linux/iscsi.go

```diff
@@ -501,6 +513,18 @@ func validateChapUserPassword(user []string, password []string) (chapInfo *model
 	chapInfo = &model.ChapInfo{Name: user[0], Password: password[0]}
 	return chapInfo, nil
 }
+func PerformDiscoveries(discoveryIPs []string) (a model.IscsiTargets, err error) {
```

Comment:
we should not do this. We need to check any one discovery IP which is reachable. thats good enough, it gives targets from all portals and we can login.